.github/workflows/daily-dev-bump.yaml

```diff
@@ -107,6 +107,7 @@ jobs:
           # Change github credentials back to the actions bot.
           GH_TOKEN="$ORIGINAL_GH_TOKEN"
 
+          gh pr edit $PR_URL $FLAGS --add-label "run-dcm-workflow"
           gh pr edit $PR_URL $FLAGS --add-label "autosubmit"
```

Comment:
nit: I think this can be a comma separated list:
`"autosubmit,run-dcm-workflow"`